src/bots/ama/subcommands/addEvent.js

```diff
@@ -0,0 +1,43 @@
+import client from '../../../client';
+import Event from '../models/Event';
+
+const addEvent = {
+  usage: 'Specify date then title of event',
```

Comment:
maybe out of scope of this pr: we could(should) make these types of objects, or something similar, reusable across bots to enforce this pattern of defining subCommand actions and handlers

Reply:
@jackwherry you should take a look and see what you think about this pattern since I think you're also working on one of the bigger sub bots. of course, we don't have to deal with it right now

Reply:
Yeah I kinda used this as an opportunity to play around with the idea of normalized subcommands. Think this is on the right track.

Reply:
I think this design makes a lot of sense for the FAQ bot. I'll start with something very similar and then work from there